test/acceptance/features/environment.py

```diff
@@ -39,9 +39,12 @@ def before_all(_context):
     if start_sbo == "local":
         assert not os.getenv("TEST_ACCEPTANCE_SBO_STARTED").startswith("FAILED"), "TEST_ACCEPTANCE_SBO_STARTED shoud not be FAILED."
     elif start_sbo == "remote":
-        sbo_namespace = os.getenv("SBO_NAMESPACE")
-        assert (sbo_namespace is not None) and (sbo_namespace != ""), f"SBO_NAMESPACE is required but it is not set or it is empty: {sbo_namespace}"
-        _context.sbo_namespace = sbo_namespace
+        output, code = cmd.run(
+            f"{ctx.cli} get deployment --all-namespaces -o json"
+            + " | jq -rc '.items[] | select(.metadata.name == \"service-binding-operator\").metadata.namespace'")
+        assert code == 0, f"Non-zero return code while trying to detect namespace for SBO: {output}"
+        assert (output is not None) and (output != ""), "Unable to find SBO's deployment in any namespace."
```

Comment:
can `output` really be `None` here, given that we execute external command? Do we wanto trim the output perhaps?

Reply:
+1